tests/integration/targets/mso_schema_site_l3out/tasks/main.yml

```diff
@@ -100,14 +101,14 @@
     <<: *mso_info
     schema: '{{ mso_schema | default("ansible_test") }}'
     template: Template1
-    l3out: '{{item.l3out}}'
+    l3out: '{{ item }}'
     vrf:
         name: VRF1
     state: present
   register: add_l3out
   loop:
-  - { l3out: L3out1}
-  - { l3out: L3out2}
+  - "{{ l3outs[0] }}"
+  - "{{ l3outs[1] }}"
```

Comment:
```
loop: {{ l3outs }}
```
Isn't this enough?

Reply:
will give it another try, didn't have success with similar attempt to do this as it was passing in the entire list and putting that as the name of the l3out

Reply:
see I had '' which messed up the loop with just l3outs, will change to loop: {{ l3outs }}